src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -379,6 +364,18 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
+/**
+ * Reload feedback response comments after adding/editing/deleting.
```

Comment:
Try not to mention where a function is used. The function exists to perform a certain function, and it doesn't matter where it is called from.